plugins/woocommerce/src/Database/Migrations/CustomOrderTable/PostsToOrdersMigrationController.php

```diff
@@ -90,6 +90,20 @@ public function migrate_orders( array $order_post_ids ): void {
 		// TODO: Return merged error array.
 	}
 
+	/**
+	 * Verify whether the given order IDs were migrated properly or not.
+	 *
+	 * @param array $order_post_ids Order IDs.
+	 *
+	 * @return array Array of failed IDs along with columns.
+	 */
+	public function verify_migrated_orders( array $order_post_ids ): array {
+		return $this->order_table_migrator->verify_migrated_data( $order_post_ids ) +
+			$this->billing_address_table_migrator->verify_migrated_data( $order_post_ids ) +
+			$this->shipping_address_table_migrator->verify_migrated_data( $order_post_ids ) +
+			$this->operation_data_table_migrator->verify_migrated_data( $order_post_ids );
```

Comment:
yeah, I plan to add meta migration ready. initially, I left it out since it is mostly a copy-paste, but it would be good to have verification logic. I will add after first round of public testing.

Reply:
> I will add after first round of public testing.

I actually think a lot of extension authors might want to test the migration to see if their data got migrated correctly, so either we should explicitly and clearly say this doesn't check custom meta data, or, ideally, add this before the public testing.